docs/basics.md

```diff
@@ -47,6 +47,7 @@ Install pre-requisites for building cardano node and using CNTools
 -c    Install/Upgrade and build CNCLI with RUST
 -w    Install/Upgrade Vacuumlabs cardano-hw-cli for hardware wallet support
 -o    Install/Upgrade Ogmios Server binary
+-x    Install/Upgrade Cardano Signer (https://github.com/gitmachtl/cardano-signer)
```

Comment:
IMO - we shouldnt have links on `-h`, as we dont for cardano-hw-cli/ogmios/cncli too.

Instead, we could potentially add this to [Build overview](https://cardano-community.github.io/guild-operators/build/) and have a brief couple of liners about each additional component, linking to their repos - ofcourse, if folks who've built the tools dont have a concern

Reply:
Apologies for delay in adding above mentioned Build Overview, done now